### ISSUE TITLE
Strange designer behaviour at the model level

### ISSUE THREAD

**Issue by alprinias** [msawczyn/EFDesigner#218]
![wrong](https://user-images.githubusercontent.com/8612300/93775327-f0fa8f80-fc2a-11ea-8eeb-555a60eedc76.JPG)
![error](https://user-images.githubusercontent.com/8612300/93775366-fbb52480-fc2a-11ea-83f6-142b9bc5ee1a.JPG)

Hi Michael,
I have a model for a couple of months now and everything was working perfectly till an hour ago. I don't know what might have happened, but now when i right-click on the designer surface I get a context menu with many more items, all enabled (none grayed out), which seem to be a mixture of items I would have for the model level and the entity level. I mean, I can see "Add Properties Via Code", which I would expect to see only if I had an entity selected on the diagram. And no matter what selection I make, I get an "Object reference was not set to an instance of an object" error.  This is with 2.0.5.3 and the designer in another project works as expected. So, tt looks like something got corrupted in the model?

Does this ring a bell? Any ideas? Or how could I start clean?  
I am attaching the images of the context menu and the error popup.
Best regrads,
Alex

**Comment by msawczyn** [comment]
Wow. Can't even fathom a way that that could happen. If you could send me the .efmodel file, I'll see if I can find out what's wrong, if it's in there. The only other thing I could suggest is to uninstall/reinstall the extension. You can send to michaelsawczyn [at] gmail[dot]com

**Comment by alprinias** [comment]
Hi Michael,
It fixed itself yesterday. I had checked the model text xml and it was perfectly alright. Then I noticed a yellow bar over the designer, saying something about the resolution, XAML/WPF etc. Then I remembered that I had recently added a registry key, following this 
https://docs.microsoft.com/en-us/visualstudio/designers/disable-dpi-awareness?view=vs-2019
in order to fix my form designer problems since I have a high resolution monitor and everything looked very small. I restarted VS by pressing on the restart link on the yellow bar and it worked! Without any visual changes in the model designer or the form designer. Strange!!!
So, it seems it was a VS designer issue and not model-related. 
I'll keep you informed if it happens again.
Best regards,
Alex
